### DISCUSSION TITLE
Ability to change `IS_COOP` placeholder output

### DISCUSSION THREAD

**Discussion by ItsFantom** [BG-Software-LLC/SuperiorSkyblock2#1016] [Implemented Feature Requests] [answered]
Currently it's only Displays `Yes` or `No` and it would be cool to be customizable

**Comment by OmerBenGera** [comment, marked as answer]
You can do that with PAPI already.

**Comment by OmerBenGera** [comment]
You can now change the output of the placeholder in the lang file.
